src/lambert-coordinaat.js

```diff
@@ -0,0 +1,75 @@
+
+/**
+ * Lambert-coördinaat reguliere expressie. Staat toe:
+ * - optioneel ongelimiteerd aantal spaties voor de start van het coördinaat
+ * - optioneel linker haakje '('
+ * - verplichte x-coördinaat, welke een decimaal of geheel getal kan zijn met 1-6 getallen voor de komma en een ongelimiteerd aantal na
+ * - verplicht scheidingsteken, welke een komma ',' of puntkomma ';' kan zijn
+ * - optionele ongelimiteerd aantal spaties
+ * - verplichte y-coördinaat, welke een decimaal of geheel getal kan zijn met 1-6 getallen voor de komma en een ongelimiteerd aantal na
+ * - optioneel rechter haakje ')'
+ *
+ * E.g.:
+ * - "104719.27, 192387.25", "104719.27,192387.25", "104719.27; 192387.25"
+ * - "(104719.27, 192387.25)", "104719.27, 192387.25)", "(104719.27, 192387.25"
+ * - "104719, 192387"
+ *
+ * @type {RegExp}
+ */
+const REGEX = /^\s*\(?(?<x>\d{1,6}\.\d{1,2}|\d{1,6})\d*[,;]\u0020*(?<y>\d{1,6}\.\d{1,2}|\d{1,6})\d*\)?/;
```

Comment:
Blijkt misschien nog uit de verdere code review, maar zou overwegen dit als static onderdeel van `LambertCoordinaat` toe te voegen of als private const behandelen.

Reply:
Blijkbaar puur om de REGEX te kunnen unit testen. Ik zou persoonlijk alleen lambert-coordinaat testen, de achterliggende REGEX wordt op die manier indirect getest. Deze REGEX moet dan niet geëxporteerd worden.